Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -118,6 +123,11 @@ public void Commit()
                     MeshObject.SetTriangles(tris, 0);
                     MeshObject.RecalculateNormals();
                     MeshObject.RecalculateBounds();
+                    if (Collider)
```

Comment:
I'll rename it to SpatialCollider as suggested above and also make this change. To be honest, I would actually prefer not even performing this test. Ideally, we want to ensure this member is always set and having it blow up with a null pointer exception would quickly draw attention to erroneous usage.